inventory/any.cf

```diff
@@ -41,6 +41,11 @@ bundle agent inventory_autorun
       "packages_refresh" usebundle => cfe_autorun_inventory_packages(),
       handle => "cfe_internal_autorun_inventory_packages";
 
+    !disable_inventory_policy_servers::
```

Comment:
do we need docs about these classes for `disable_inventory_*`?

Reply:
https://docs.cfengine.com/docs/3.15/reference-masterfiles-policy-framework-inventory.html

> The inventory_control bundle lives in def.cf and defines what inventory modules should be disabled. You can simply set disable_inventory to avoid the whole system, or you can look for the disable_inventory_xyz class to disable module xyz.
￼